diff --git a/node_modules/react-native-rate/android/build.gradle b/node_modules/react-native-rate/android/build.gradle
index 5366d92..7741c7b 100644
--- a/node_modules/react-native-rate/android/build.gradle
+++ b/node_modules/react-native-rate/android/build.gradle
@@ -29,7 +29,7 @@ buildscript {
     if (project == rootProject) {
         repositories {
             google()
-            jcenter()
+            mavenCentral()
         }
         dependencies {
             classpath 'com.android.tools.build:gradle:3.4.1'
@@ -66,7 +66,7 @@ repositories {
         url "$rootDir/../node_modules/jsc-android/dist"
     }
     google()
-    jcenter()
+    mavenCentral()
 }
 
 dependencies {
